selfdrive/locationd/laikad.py

```diff
@@ -136,10 +136,11 @@ def read_report(self, gnss_msg):
     return week, tow, new_meas
 
   def is_ephemeris(self, gnss_msg):
+    mwhich = gnss_msg.which()
     if self.use_qcom:
-      return gnss_msg.which() == 'drSvPoly'
+      return mwhich == 'drSvPoly'
     else:
-      return gnss_msg.which() == 'ephemeris'
+      return mwhich == 'ephemeris' or mwhich == 'glonassEphemeris'
 
   def read_ephemeris(self, gnss_msg):
     # TODO this only works on GLONASS
```

Comment:
does this comment have to be updated? not sure what's it's referring to

Reply:
yea, it doesnt make sense, removed it